rosdep/python.yaml

```diff
@@ -2890,6 +2899,9 @@ python-parameterized:
     artful:
       pip:
         packages: [parameterized]
+    focal:
+      pip:
+        packages: [parameterized]
```

Comment:
According to https://packages.ubuntu.com/focal/python3-parameterized version 0.7.0 is packaged in Ubuntu Focal. Would now be a good time to add a `python3-parameterized` key which uses the system dependency instead of relying on pip keys?

Reply:
Do you suggest adding an entry like this instead?
```
python3-parameterized:
  ubuntu:
    focal: [python3-parameterized]
```

Reply:
yes, we should use the debian packaged version if it exists. pip can be used to backfill necessary platforms where the debian package doesn't exist.